source/brailleDisplayDrivers/albatross/driver.py

```diff
@@ -484,9 +484,9 @@ def _skipRedundantInitPackets(self, data: bytes) -> bool:
 							# for a braille display driver
 							"To use Albatross with NVDA: "
 							"change number of status cells in Albatross internal menu at most "
-							f"to {MAX_STATUS_CELLS_ALLOWED}, and if needed, restart Albatross "
+							"to {maxCells}, and if needed, restart Albatross "
 							"and NVDA.",
```

Comment:
```suggestion
							"To use an Albatross with NVDA, change the number of status cells in the "
							"Albatross's internal menu to at most {maxCells}, and restart the Albatross and "
							"NVDA if needed."
```

Reply:
I have taken your wording suggestion.

FYI I have not been able to apply your suggestion direction; GitHub says me:
> Applying suggestions on deleted lines is currently not supported.Refresh and try again.

So I have put it manually in commit b22e96cd52f6db94d525a45ac3dfa9ba3524fc2c.